### ISSUE TITLE
roblox studio: "cannot load place due to permission error"

### ISSUE THREAD

**Issue by sarenord** [roblox-linux-wrapper/roblox-linux-wrapper#171]
when I try and launch a place in studio, it loads for a second, then gives me a black screen where the place whould be and throws the error "cannt load place due to permission error. please check permissions and try again". anyone else having this problem?

**Comment by firefish5000** [comment]
I cant help you, but I had that problem at one point in time. couldn't even load the base-plate or flat terrain worlds. But I haven't had that problem since I started using it on my gentoo system (never got arch working with it). If different versions don't work, I also granted all wine binaries CAP_NET_RAW.

Not sure if that capability helped at all, since that was during my attempts to make the player work, not the studio. Never was able to get in a game for more than a few seconds. though I did manage to load the map, spawn, and see other players at spawn (all frozen).

FWIW, the studio seems to be 100% functional to me. I have not only created local worlds, but also edited places I saved/uploaded from other machines, joined team edit session on a world owned by another developer, and played/debugged the world with the other developer in the studio's online test server mode. The last of which really makes me wonder why the player wont work....
